core-request.html

```diff
@@ -141,17 +141,19 @@
       }.bind(this));
 
       xhr.open(
-        options.method,
+        options.method || 'GET',
         options.url,
-        options.async
+        options.async !== false
```

Comment:
:crying_cat_face: how about just hardcode this to `false`. Sync XHR is bad and you should feel bad for enabling it. :P